.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+---
+contact_links:
+  - name: Ask a question or get support right here
```

Comment:
We don't provide "support" and we shouldn't create the expectation that people will be able to receive support (which usually has connotations around responsiveness, SLAs, etc. and is associated with commercial ventures). We could change it to "or get help".